### DISCUSSION TITLE
API expose from Budibase

### DISCUSSION THREAD

**Discussion by GarryXian** [Budibase/budibase#4647] [Help]
Hi 

We wonder if budibase allows developer to build and expose API from Budibase application to other micro services?

**Comment by SantoshSrinivas79** [comment]
It's undocumented but the API is pretty good. You can use developer tools or a proxy tool to check the endpoints

**Comment by shogunpurple** [comment]
Hi @GarryXian 

As @SantoshSrinivas79 correctly states, we have our internal REST API available and you can find the endpoints through some digging in the Network panel. There are some discussions on this forum too that show it in use - however, we are currently working on the official, documented budibase REST API, and it will be shipped in the next 2 weeks. This will provide all of the features you need and also include better security, cleaner endpoints and URLs, as well as full API documentation with swagger support.
